scripts/grpc-integration-test.sh

```diff
@@ -1,105 +0,0 @@
-#!/bin/bash
```

Comment:
why is this file deleted in this PR? Does not seem relevant.

Reply:
yup, this is unrelated to this PR. i will create a separate PR.

Reply:
#5372